### PR TITLE
upgrade to monaco-editor 0.49.0

### PR DESCRIPTION
**Pull request by morgilad**
Upgrade to monaco-editor 0.49.0

### CODE REVIEW

**Review by Weffe — DISMISSED** [review]
LGTM

File: yarn.lock
Comment (maxburs) [inline yarn.lock:8044]: Patch needs to be updated?

Either:
- Patch is still required, and we need to update it. This also implies the CI script to check if the samples work is broken 😞
- Patch is no longer required, and we can delete it! 🎊

Comment (morgilad) [reply yarn.lock:8044]: What issue does this patch resolve? How can I verify it's working as expected? 
I checked and saw that my local environment is functioning well, but I haven't tested it locally on KustoWeb or Trident due to the linking issue we discussed. 
Additionally, if the 'test samples' action passed, doesn't that indicate it's not broken?

Comment (morgilad) [reply yarn.lock:8044]: I’ve updated the patch file. 
However, I’m still uncertain why the samples test doesn’t result in a failure on GitHub Actions.
@maxburs waiting for your response before merging to master.

Comment (maxburs) [reply yarn.lock:8044]: > What issue does this patch resolve? How can I verify it's working as expected?

The Parcel samples will be broken if the patch isn't applied and is still necessary. The patch contains a link to the relevant Parcel bug, and the samples README's contain more details.

> However, I’m still uncertain why the samples test doesn’t result in a failure on GitHub Actions.

I'd guess there's a mistake in the script. I had a lot of problems trying to get it to start/stop each sample. Sometimes, a previous sample wouldn't stop, and the script would re-test the same sample multiple times.

Comment (maxburs) [reply yarn.lock:8044]: Problem spot is here:

/scripts/test-samples.ts
image

**Pull request merged**
